### ISSUE TITLE
Add the ability to have MFA authentication - for Azure DBs

### ISSUE THREAD

**Issue by BP-vismaNmbrs** [ErikEJ/SqlQueryStress#142]
Currently we only have the "Integrated Authentication" and "SQL Server Authentication" but sometimes there is also the use of MFA, specially to prevent a SQL user to be added.
Something on the line of [this](https://stackoverflow.com/questions/56215700/connecting-to-azure-db-with-active-directory-universal-with-mfa-support-authenti) .

**Comment by ErikEJ** [comment]
@BP-vismaNmbrs  Have you tried the latest build? https://github.com/ErikEJ/SqlQueryStress/releases/download/102/SqlQueryStressNet6.zip

**Comment by ErikEJ** [comment]
duplicate of #113

**Comment by BP-vismaNmbrs** [comment]
Oh... what a rookie mistake. I'm sorry for this and thanks for pointing it out @ErikEJ

**Comment by ErikEJ** [comment]
@BP-vismaNmbrs No problem, let me know if it works for you

**Comment by ErikEJ** [comment]
Works on my PC - I am using Edge as my default browser

**Comment by ErikEJ** [comment]
1: Enter name of server: abc.database.windows.net
2: Pick MFA from combo
3: enter login
4: Select database combobox and login process occurs

**Comment by BP-vismaNmbrs** [comment]
Today was the day I found some time to go over this on my machine.

Following up on this, I managed to get it working but with some workaround - maybe due to my network security policies.

I'm leaving the steps here in case it helps someone else:
1. Downloaded the latest version that @ErikEJ posted in this thread
2. Unzipped and run the .exe
3. Filled in the database popup with the correct values and as soon as I filled in my login name the azure login appeared on the browser
4. I filler in my credentials and it redirected to "localhost....(long code and stuff)" and the application got stuck on wait
5. The workaround is to use Azure CLI and login to azure with the `az login --use-device-code` command
6. Follow the instructions and the SQL QueryStress application get's the response back and works fine

I found the solution thanks to this link, after several interactions:
[az login broken on chrome with ERR_SSL_PROTOCOL_ERROR](https://github.com/Azure/azure-cli/issues/10426#issuecomment-528308713)
